### PR TITLE
Add function to pause execution for a specified time

### PR DESCRIPTION
**Pull request by PlaceReporter99**
now use `(num) ¨w` to wait num seconds!

### CODE REVIEW

**Review by nayakrujul — COMMENTED** [review]
Looks good, but do you want to add a check to see if it's a number so that `time.sleep` doesn't error?

**Comment by PlaceReporter99** [comment]
at some point

**Review by Lyxal — CHANGES_REQUESTED** [review]
I think adding a check is a good idea

**Comment by PlaceReporter99** [comment]
> I think adding a check is a good idea

@Lyxal @nayakrujul  fixed

File: vyxal/elements.py [hunk 1]
```diff
@@ -443,6 +444,7 @@ def process_element(
     "k□": process_element("[[0,1],[1,0],[0,-1],[-1,0]]", 0),
     "kṘ": process_element('"IVXLCDM"', 0),
     "k•": process_element('["qwertyuiop","asdfghjkl","zxcvbnm"]', 0),
+    "¨w": ("time.sleep((_:=pop(stack, 1, ctx)) if type(_)==type(1) else 0)", 1),
```
Comment (Lyxal) [inline vyxal/elements.py:447]: This shouldn't push 0. How about `lhs = pop(stack, 1, ctx)\nif vy_type(lhs) == NUMBER_TYPE: time.sleep(int(lhs))`

**Review by Lyxal — CHANGES_REQUESTED** [review]

File: vyxal/elements.py (hunk 1 above)
Comment (PlaceReporter99) [reply vyxal/elements.py]: It waits 0

**Review by Lyxal — APPROVED** [review]

**Review by nayakrujul — APPROVED** [review]

**Pull request merged**